ProjectSidewalk/SidewalkWebpage
# Resolves #1281, #1288: Add tagging to tutorial and update gifs

Resolves #1281 
Resolves #1288 

This pr updates the tutorial to include the tagging feature for labels and also updates the gifs in the tutorial to be up-to-date with the current label UI. 

To test:
- Step through the tutorial to see if the gifs look good and if everything works as intended.
- Make sure to run through every branch of the tutorial (click on the wrong severity ratings and tags).
- Check the progress bar on the right side of the screen to make sure it increments as one would expect it to. 
- Check out the shiny new gifs. Let me know if you have any feedback on them. They're pretty simple to change.

@misaugstad We never iterated through possible texts for the MissingSidewalk tutorial screen. Let me know what you think of the text that I put there. 


I tested this on the current develop branch with no issues.
